sparkify/transformers/outlier_detection/outlier_mad_helpers.py

```diff
@@ -0,0 +1,16 @@
+from pyspark.sql.functions import abs
+from pyspark.sql.functions import lit
+
+
+def mark_outliers_mad(dataset, col_name):
```

Comment:
Import the relevant pyspark dataframe and add the following type hinting:
```suggestion
def mark_outliers_mad(dataset: DataFrame, col_name: str) -> DataFrame:
```